toolflow/boot/misc/resizefs

```diff
@@ -0,0 +1,14 @@
+sudo fdisk /dev/mmcblk0 <<EOF
```

Comment:
Is this a stable API?

Reply:
it is used by raspi-conf, so I assume it